src/System.Private.CoreLib/shared/Interop/Windows/BCrypt/Interop.BCryptGenRandom.GetRandomBytes.cs

```diff
@@ -0,0 +1,29 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    internal static unsafe void GetRandomBytes(byte* buffer, int length)
+    {
+        Debug.Assert(buffer != null);
+        Debug.Assert(length >= 0);
+
+        BCrypt.NTSTATUS status = BCrypt.BCryptGenRandom(IntPtr.Zero, buffer, length, Interop.BCrypt.BCRYPT_USE_SYSTEM_PREFERRED_RNG);
```

Comment:
Nit: `Interop.` is not necessary.

Reply:
ah ok...i thought it was good for clarity, no problem.